CHANGELOG.md

```diff
@@ -126,6 +126,8 @@
   [#2457](https://github.com/Kong/kong-operator/pull/2457)
 - Add support to HTTPRoute RequestRedirect filter
   [#2470](https://github.com/Kong/kong-operator/pull/2470)
+- Add CLI flag `--enable-fqdn-mode` to indicate whether to use FQDN endpoints for service discovery.
```

Comment:
We might want to be a bit more descriptive what exactly this option does. WDYT?

Reply:
449d48ce00886fbf190a0f9e7362cba224f83ea7